relay-server/Cargo.toml

```diff
@@ -22,6 +22,7 @@ processing = [
     "relay-redis/impl",
     "symbolic",
 ]
+jsonschema = ["relay-general/jsonschema"]
```

Comment:
Do we need the feature on `relay-server`? It looks like the server doesn't need this anywhere and it would be great if we could go lengths to remove this from the server.

Reply:
I fixed it, not sure if that's the way it's supposed to be done